views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+<div>
+    {{#each beersFromApi}}
```

Comment:
The name of the property you're sending from `app.js` is `randomBeer` here so that's the one you should be using
```
{{each randomBeer}}